.github/workflows/release-prod.yml

```diff
@@ -0,0 +1,59 @@
+name: Release Production App
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_and_test:
+    uses: ./.github/workflows/build.yml
+    with:
+      storeArtifacts: true
+    secrets: inherit
+
+  prod_release_ios:
+    name: Upload iOS build to App Store connect
+    runs-on: macos-latest
+    needs: [build_and_test]
+    if: github.ref_name == 'production'
```

Comment:
This condition will not be made when we are triggered by a new release

Reply:
Thanks for catching this